ui/src/schemas/index.ts

```diff
@@ -43,3 +43,15 @@ export const ortRunStatus: z.ZodEnum<[OrtRunStatus, ...OrtRunStatus[]]> =
 export const statusSchema = z.object({
   status: z.array(ortRunStatus).optional(),
 });
+
+// Enum schema for the possible values of the issue severities
+export const issueSeverity: z.ZodEnum<[Severity, ...Severity[]]> = z.enum([
+  'HINT',
+  'WARNING',
+  'ERROR',
+]);
+
+// Issue severity schema that is used for search parameter validation
+export const issueSeveritySchema = z.object({
+  severity: z.array(issueSeverity).optional(),
+});
```

Comment:
More idiomatic zod naming would be that `issueSeverity` is `issueSeveritySchema` and the current `issueSeveritySchema` something different.

Reply:
I mimicked the naming from Johanna's definitions earlier in that file, so they are in line with each other.